contents/tutorials/experiments.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to use Experiments without feature flags
+sidebar: Docs
+showTitle: true
+featuredTutorial: false
+author: ['neil-kakkar']
+topics: ['feature flags']
+date: 2021-05-11
+---
+
+Say you want to use Experiments, but not the feature flag library, because you've rolled your own. Or, more commonly, say feature flag support in your favourite PostHog client library doesn't yet exist.
```

Comment:
```suggestion
Say you want to use Experiments, but not the feature flag library, because you've rolled out your own. Or, more commonly, say feature flag support in your favourite PostHog client library doesn't yet exist.
```